### PR TITLE
Added list of knowledge articles to response page

### PR DESCRIPTION
**Pull request by alistairknight**
# **What this PR does / why we need it**:
*A brief description of changes being made*
*Link to the relevant work items: e.g: Relates to ADO Work Item AB#213700 and builds on #3376 (link to ADO Build ID URL)*

# **Special notes for your reviewer**
*Any specific actions or notes on review?*

# Testing
*Any relevant testing information and pipeline runs.*

# Checklist (please delete before completing or setting auto-complete)
- [ ] Story Work items associated (not Tasks)
- [ ] Successful testing run(s) link provided
- [ ] Title pattern should be `{work item number}: {title}`
- [ ] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests
- [ ] Version number in **package.json** has been updated

# **How does this PR make you feel**:
gif

### CODE REVIEW

**Review by sfitz42 — CHANGES_REQUESTED** [review]
Just a few small things.

File: app/config/api.js [hunk 1]
```diff
@@ -13,6 +16,9 @@ const config = {
   documentsApi: {
     baseUrl: process.env.DOCUMENTS_API_BASE_URL
   },
+  knowledgeApi: {
+    baseUrl: process.env.KNOWLEDGE_API_BASE_URL
```
[the remark this answers is not in the document]
Comment (alistairknight) [reply app/config/api.js:20]: Added

File: app/routes/document-configure-response.js [hunk 2]
```diff
@@ -8,8 +9,9 @@ module.exports = [{
     handler: async (request, h) => {
       const documentId = request.params.id
       const contents = await getDocumentContent(documentId)
+      const knowledge = await getAllKnowledge('')
```
[the remark this answers is not in the document]
Comment (alistairknight) [reply app/routes/document-configure-response.js]: Sorted

File: app/views/document-configure-response.njk [hunk 3]
```diff
@@ -89,10 +89,40 @@
...
+          </div>
+          <div class="govuk-grid-column-one-third">
+            <strong>Date</strong>
+          </div>
+          <div class="govuk-grid-column-one-third">
+            <strong>Category</strong>
+          </div>
+        </div>
+
+        {% for k in knowledge %}
```
[the remark this answers is not in the document]
Comment (alistairknight) [reply app/views/document-configure-response.njk]: Great point

**Review by sfitz42 — APPROVED** [review]
LGTM

**Pull request merged**